networkx/algorithms/dag.py

```diff
@@ -628,9 +629,9 @@ def is_aperiodic(G):
 
     Notes
     -----
-    This uses the method outlined in [1]_, which runs in $O(m)$ time
-    given $m$ edges in `G`. Note that a graph is not aperiodic if it is
-    acyclic as every integer trivial divides length 0 cycles.
+    This uses the method outlined in [1]_, which runs in $O(n + m)$ time
+    given $n$ nodes and $m$ edges in `G`. Note that a graph is not aperiodic
+    if it is acyclic as every integer trivially divides length 0 cycles.
```

Comment:
This should probably be corrected to talk about "every cycle" when there are none, rather than cycles of length 0. Acyclic does not mean cycles of length 0.

Reply:
One possible formulation:
```suggestion
    if it is acyclic, as every integer divides all cycles (because there are no cycles to divide).
```